src/main/java/Wolox/training/exceptions/BookAlreadyOwnedException.java

```diff
@@ -0,0 +1,11 @@
+package Wolox.training.exceptions;
+        import java.lang.RuntimeException;
+        import org.springframework.http.HttpStatus;
+        import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.UNPROCESSABLE_ENTITY)
+public class BookAlreadyOwnedException extends  RuntimeException{
+    public BookAlreadyOwnedException() {
+        super("You already have this associated book");
```

Comment:
This can be a constant
```suggestion
        super(MESSAGE);
```